kyuubi-events/src/main/scala/org/apache/kyuubi/events/EventLoggerType.scala

```diff
@@ -21,6 +21,5 @@ object EventLoggerType extends Enumeration {
 
   type EventLoggerType = Value
 
-  // TODO: Only SPARK is done now
-  val SPARK, JSON, JDBC, CUSTOM = Value
+  val SPARK, JSON, JDBC, CUSTOM, KAFKA = Value
```

Comment:
all others is supported now?

Reply:
Seems JDBC is not supported.

Reply:
I think this TODO can be removed. Not all the event logger types in both servers or the engine. It's a legacy todo hints.